docker/Dockerfile.loongarch64-unknown-linux-gnu

```diff
@@ -15,7 +15,7 @@ FROM cross-base as build
 ARG VERBOSE
 COPY crosstool-ng.sh /
 COPY crosstool-config/loongarch64-unknown-linux-gnu.config /
-RUN /crosstool-ng.sh loongarch64-unknown-linux-gnu.config 5
+RUN /crosstool-ng.sh loongarch64-unknown-linux-gnu.config 5 ed12fa68402f58e171a6f79500f73f4781fdc9e5
```

Comment:
Is there no tagged release?

Reply:
Not yet. `rust-lang/rust` also uses the git version to build loongarch64 target:
https://github.com/rust-lang/rust/blob/master/src/ci/docker/scripts/crosstool-ng-git.sh
https://github.com/rust-lang/rust/blob/5db9d432070087c13330e85221010671174025d5/src/ci/docker/host-x86_64/dist-loongarch64-musl/Dockerfile#L6